src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js

```diff
@@ -235,7 +235,7 @@ const OncoGridWrapper = compose(
       const performanceContext = {
         donors: responses.cases.length,
         genes: responses.genes.length,
-        occurences: responses.occurences.length,
+        occurences: responses.occurrences.length,
```

Comment:
Existing metrics were gathered as `occurences`, want to keep the data consistent